python/runtime/utils/PyRemoteSimulatorQPU.cpp

```diff
@@ -74,6 +74,41 @@ static void _launchKernel(cudaq::ExecutionContext *executionContextPtr,
     throw std::runtime_error("Failed to launch kernel. Error: " + errorMsg);
 }
 
+static void
+_launchKernelStreamline(cudaq::ExecutionContext *executionContextPtr,
```

Comment:
nit: shouldn't really start library implementation functions with an `_`.

Reply:
Good point. I renamed all of those functions to use the `Impl` suffix instead.